### PR TITLE
CI: Rename exercise jobs & updates

### PR DESCRIPTION
**Pull request by SaschaMann**

### CODE REVIEW

File: .github/workflows/exercise-tests.yml [hunk 1]
```diff
@@ -20,8 +21,7 @@ jobs:
     steps:
       - uses: actions/checkout@v1.0.0
```
Comment (SaschaMann) [inline .github/workflows/exercise-tests.yml:22]:
```suggestion
      - uses: actions/checkout@v2
```

File: .github/workflows/exercise-tests.yml [hunk 2]
```diff
@@ -10,6 +10,7 @@ on:
 
 jobs:
   test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
```
Comment (SaschaMann) [inline .github/workflows/exercise-tests.yml:13]:
```suggestion
    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ github.event_name }}
```

**Comment by SaschaMann** [comment]
I noticed while reviewing #280 that the names weren't as descriptive as the standard PkgTemplates job names but it didn't let me update them via a comment

**Review by cmcaine — APPROVED** [review]

File: .github/workflows/exercise-tests.yml [hunk 3]
```diff
@@ -10,6 +10,7 @@ on:
 
 jobs:
   test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ github.event_name }}
```
Comment (SaschaMann) [inline .github/workflows/exercise-tests.yml:13]:
```suggestion
    name: Julia ${{ matrix.julia-version }} - ${{ matrix.os }} - ${{ github.event_name }}
```

**Comment by SaschaMann** [comment]
Thanks for the fast review :)

**Pull request merged**